src/framework/uicomponents/view/dialogview.cpp

```diff
@@ -118,8 +121,9 @@ void DialogView::updateGeometry()
     setContentWidth(dlgRect.width());
     setContentHeight(dlgRect.height());
 
-    //! NOTE ok will be if they call accept
-    setErrCode(Ret::Code::Cancel);
+    if (m_window) {
+        m_window->show(resolveScreen(), dlgRect, m_openPolicy != OpenPolicy::NoActivateFocus);
+    }
```

Comment:
I don't quite understand this change. The window is already visible at this point, isn't it? If you're calling this purely to adjust the geometry, then it would be better to create a dedicated method for that, instead of abusing the `show` method.